corporate/views/installation_activity.py

```diff
@@ -223,14 +221,9 @@ def realm_summary_table() -> str:
                 row["arr"] = f"${cents_to_dollar_string(estimated_arrs[string_id])}"
 
             if row["plan_type"] in [Realm.PLAN_TYPE_STANDARD, Realm.PLAN_TYPE_PLUS]:
-                row["effective_rate"] = 100 - int(realms_with_default_discount.get(string_id, 0))
+                row["effective_rate"] = plan_rates.get(string_id, "")
             elif row["plan_type"] == Realm.PLAN_TYPE_STANDARD_FREE:
                 row["effective_rate"] = 0
-            elif (
-                row["plan_type"] == Realm.PLAN_TYPE_LIMITED
-                and string_id in realms_with_default_discount
-            ):
-                row["effective_rate"] = 100 - int(realms_with_default_discount[string_id])
```

Comment:
Would it make sense to still display an effective rate for this case based on their `required_plan_tier` configuration? 

Could be a follow-up, but that seems to make sense to me.